plane/source/docs/quadplane-frame-setup.rst

```diff
@@ -7,16 +7,15 @@ QuadPlane Frame setup
 The QuadPlane code supports several frame arrangements of quadcopter,
 hexacopter, octacopter and octaquad multicopter frames which use lifting motors in addition to the normal forward motor(s). Also configurations in which the VTOL motors tilt for transitions or control, as well as choice between horizontal VTOL stance or vertical (Tailsitters).
 
-
 Tailsitters
 ===========
 
 Frame setup for Tailsitters is in this section: :ref:`guide-tailsitter`. Once the frame is configured, proceed with the other :ref:`quadplane-setup` instructions.
 
-Plane VTOL Stance
-=================
+Plane VTOL Motor Configurations
+===============================
 
-These configurations add multicopter style lifting motors to a conventional fixed wing configuration. Some or all of these motors may also be configured as tilting motors to be used in fixed wing flight.
+These configurations add multicopter style lifting motors to a conventional fixed wing configuration. Some or all of these motors may also be configured as tilting motors to be used in fixed wing flight instead of the normal fixed forward motor(s).
 
 Frame Types and Classes
```

Comment:
I think we should include the note about rebooting required no?

.. note:: You should reboot the autopilot after setting :ref:`Q_FRAME_CLASS<Q_FRAME_CLASS>` and/or
 :ref:`Q_FRAME_TYPE<Q_FRAME_TYPE>` and before making any other changes as they will change the default assignment of output channel.

Reply:
yep missed adding that...will do